File: Go/fizzbuzz.go
package main

import (
	"fmt"
)

func main() {
	fizzbuzz(100)
}

func fizzbuzz(upto int) {
	for num := 1; num < upto+1; num++ {
		if num%3 == 0 && num%5 == 0 {
			fmt.Println("FizzBuzz")
		} else if num%3 == 0 {
			fmt.Println("Fizz")
		} else if num%5 == 0 {
			fmt.Println("Buzz")
		} else {
			fmt.Println(num)
		}
	}
}

File: C/fizzbuzz.c
#include <stdio.h>

int fizzBuzz(int upto)
{
    for (int num = 1; num < upto + 1; num++)
    {
        if (num % 3 == 0 && num % 5 == 0)
        {
            printf("FizzBuzz\n");
        }
        else if (num % 3 == 0)
        {
            printf("Fizz\n");
        }
        else if (num % 5 == 0)
        {
            printf("Buzz\n");
        }
        else
        {
            printf("%d \n", num);
        }
    }
    return 0;
}

int main()
{
    fizzBuzz(100);
    return 0;
}